Project-II-Calculator/calculator/src/components/ButtonComponents/Pad.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Button.css';
+import NumberButton from './NumberButton'
+
+function KeyPad() {
```

Comment:
Try to get in the habit of naming your js files the same as your components. In the long run, it makes components easier to manage when you have hundreds of them.